index.js

```diff
@@ -17,6 +17,20 @@ const commands = {
 
     comfy.Say(message, channel);
   },
+  so: (user, message, {broadcaster, mod}, {channel}) => {
+    // only broadcasters or mods can shoutout
+    // don't want the !so command to fight with any existing channel !so commands
+    // so should return if not BOTOWNER
+    if((!broadcaster && !mod) || channel !== process.env.BOTOWNER ) {
+      return;
+    }
+    const parts = message.split(" ");
+    const channel = parts.find(part => part.startsWith("@"))
```

Comment:
This variable name conflicts with the `channel` on line 20 — maybe we could call this something like `targetChannel` so it's clearer what we're doing?

Reply:
Fixed